Append -sonic to the kernel ABI, since this is a SONiC-specific kernel

From: Saikrishna Arcot <sarcot@microsoft.com>

This is being done because in SONiC, the platform-specific kernel
modules are compiled and packaged into deb files. For dependency
management purposes, those packages should depend on the SONiC-specific
kernel, since they are compiled against that kernel.
---
 debian/bin/gencontrol.py |    2 ++
 debian/control.md5sum    |    2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/bin/gencontrol.py b/debian/bin/gencontrol.py
index cde8126a1..46445907f 100755
--- a/debian/bin/gencontrol.py
+++ b/debian/bin/gencontrol.py
@@ -565,6 +565,8 @@ linux-signed-{vars['arch']} (@signedtemplate_sourceversion@) {dist}; urgency={ur
         else:
             self.abiname = f'{version.linux_upstream_full}'
 
+        self.abiname += "-sonic"
+
         self.vars = {
             'upstreamversion': self.version.linux_upstream,
             'version': self.version.linux_version,
diff --git a/debian/control.md5sum b/debian/control.md5sum
index e12b79a26..a7aec5748 100644
--- a/debian/control.md5sum
+++ b/debian/control.md5sum
@@ -1,4 +1,4 @@
-f1edf4de27267e710a88f5dc6153049a  debian/bin/gencontrol.py
+ffab9210218dc1ae03a601eed2198423  debian/bin/gencontrol.py
 c39cea595fefbe29269b0d1d0f32a6c0  debian/build/version-info
 ba2c200f8c43f7ec8d8323561739379a  debian/config/alpha/defines.toml
 9c7e42b270c37dcbc951f0696b8f4293  debian/config/amd64/defines.toml
